debian/clearwater-infrastructure.init.d

```diff
@@ -79,6 +79,15 @@ do_start()
         # Reload monit to pick up any changes to its config files
         pkill -HUP monit
 
+        # Restart the socket factories if they're running to pick up any changes
+        # in namespace configuration
+        if ( status clearwater-socket-factory-mgmt 2>&1 | grep start ); then
+          restart clearwater-socket-factory-mgmt
+        fi
+        if ( status clearwater-socket-factory-sig 2>&1 | grep start ); then
+          restart clearwater-socket-factory-sig
+        fi
```

Comment:
OK.  FTR, this is just defensive code to do a stop/restart only if we know the job is running (otherwise, "stop" rather unhelpfully fails, causing some parent scripts to bomb out)